build.gradle

```diff
@@ -64,12 +64,37 @@ subprojects {
         }
         boolean isKotlinLibrary = project.plugins.hasPlugin("org.jetbrains.kotlin.jvm") || project.plugins.hasPlugin("org.jetbrains.kotlin.android")
         if (isKotlinLibrary) {
+            java {
+                toolchain.languageVersion.set(JavaLanguageVersion.of(11))
+            }
+            kotlin {
+                jvmToolchain(11)
+            }
             it.tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
                 kotlinOptions {
                     freeCompilerArgs += extraKotlincArgs
                 }
             }
         }
+
+        boolean isAndroidLibraryOrApp = project.plugins.hasPlugin("org.jetbrains.kotlin.android")
+        if (isAndroidLibraryOrApp) {
+            android {
+                compileOptions {
+                    sourceCompatibility JavaVersion.VERSION_1_8
+                    targetCompatibility JavaVersion.VERSION_1_8
+                }
+
+            }
+        }
+
+        if (isKotlinLibrary && isAndroidLibraryOrApp) {
+            android {
+                kotlinOptions {
+                    jvmTarget = "1.8"
```

Comment:
Do we still need to target 1.8?

Reply:
Could prolly target 11 since it's tooling, but trying to limit the scope of this already large change